MobileCenter.podspec

```diff
@@ -47,5 +47,15 @@ Pod::Spec.new do |s|
       ss.vendored_frameworks = "MobileCenter-SDK-iOS/MobileCenterCrashes.framework"
   end
 
+#  s.subspec 'MobileCenterUpdates' do |ss|
```

Comment:
We need to create an internal release for the podspec and doing it all locally is simply a task of it's own and is out of scope of this task imho. I'm adding a task for this for next week.